### PR TITLE
migration: Add cases about live migration with copy storage

### PR DESCRIPTION
**Pull request by cliping**
1. VIRT-297898 - VM live migration with copy storage - network data transport - TLS - tls-destination

2. VIRT-297897 - VM live migration with copy storage- network data transport - TLS - wrong cert configurations

### CODE REVIEW

**Comment by cliping** [comment]
Test result:

 (1/8) type_specific.io-github-autotest-libvirt.migration_with_copy_storage.network_data_transport.tls.tls_destination.correct_value.copy_storage_all.p2p: PASS (294.96 s)
 (2/8) type_specific.io-github-autotest-libvirt.migration_with_copy_storage.network_data_transport.tls.tls_destination.correct_value.copy_storage_all.non_p2p: PASS (297.12 s)
 (3/8) type_specific.io-github-autotest-libvirt.migration_with_copy_storage.network_data_transport.tls.tls_destination.correct_value.copy_storage_inc.p2p: PASS (297.76 s)
 (4/8) type_specific.io-github-autotest-libvirt.migration_with_copy_storage.network_data_transport.tls.tls_destination.correct_value.copy_storage_inc.non_p2p: PASS (296.08 s)
 (5/8) type_specific.io-github-autotest-libvirt.migration_with_copy_storage.network_data_transport.tls.tls_destination.wrong_value.copy_storage_all.p2p: PASS (235.74 s)
 (6/8) type_specific.io-github-autotest-libvirt.migration_with_copy_storage.network_data_transport.tls.tls_destination.wrong_value.copy_storage_all.non_p2p: PASS (235.62 s)
 (7/8) type_specific.io-github-autotest-libvirt.migration_with_copy_storage.network_data_transport.tls.tls_destination.wrong_value.copy_storage_inc.p2p: PASS (236.77 s)
 (8/8) type_specific.io-github-autotest-libvirt.migration_with_copy_storage.network_data_transport.tls.tls_destination.wrong_value.copy_storage_inc.non_p2p: PASS (236.87 s)

File: provider/migration/base_steps.py [hunk 1]
```diff
@@ -399,3 +412,45 @@ def recreate_conn_objs(params):
     migration_base.cleanup_conn_obj(migration_obj.conn_list, migration_obj.test)
     migration_obj.conn_list.append(migration_base.setup_conn_obj(transport_type, params, migration_obj.test))
     time.sleep(3)
+
+
+def prepare_disks_remote(params, vm):
```
Comment (chunfuwen) [inline provider/migration/base_steps.py:417]: miss vm docstring

Comment (cliping) [reply provider/migration/base_steps.py:417]: Updated, thanks.

**Review by chunfuwen — APPROVED** [review]
lgtm

**Review by Yingshun — APPROVED** [review]
LGTM

**Pull request merged**